src/services/codefixes/codeFixProvider.ts

```diff
@@ -0,0 +1,48 @@
+/* @internal */
+namespace ts {
+    export interface CodeFix {
+        errorCodes: number[];
+        getCodeActions(context: CodeFixContext): CodeAction[];

```

Comment:
`getCodeActions(context: CodeFixContext): CodeAction[] | undefined` to allow strictNullChecks to work for users implementing it